server/views/pages/not-found.njk

```diff
@@ -0,0 +1,23 @@
+{% extends "../partials/layout.njk" %}
+
+{% block pageTitle %}
+  {{ locale.page.title }}
+{% endblock %}
+
+{% set mainClasses = "app-container govuk-body" %}
+
+{% block header %}
+  {% include "../partials//header.njk" %}
```

Comment:
double slash?

Reply:
Resolved in https://github.com/ministryofjustice/hmpps-sentence-plan-ui/pull/227/commits/c833e7674387179e5b647d82cdbd785fd0a901ce